### ISSUE TITLE
Train Acc 很低

### ISSUE THREAD

**Issue by yichengye310** [gaussic/text-classification-cnn-rnn#109]
您好，我今天尝试运行了一下代码，跑出来Train Acc很低，只有10%左右。参数如下：
train: 3000,
test: 500,
validate: 500,
vocab_size: 2500,
batch_size: 100
能指导一下是哪边出的问题吗？谢谢！

**Comment by liutianling** [comment]
这么少的数据你也赶上深度学习，厉害了

**Comment by StarZhai** [comment]
请问使用自己的数据进行分类时候 cnews.vocab.txt这个文件中的数据修改了么，根据什么修改的呢。我手动将train，cal，test中的数据改成自己的了，但是没动cnews.vocab.txt这个文件。我的数据量5个类乘以每个类650条数据。最终结果：
Acc:    nan%, Val Loss:    1.6, Val Acc:  21.20%,
而且也是只训练了一轮就结束了
@yeyicheng @yunhenk
